### DISCUSSION TITLE
fast way to get a best score with different option in a model

### DISCUSSION THREAD

**Discussion by lin2000y** [biolab/orange3#5842] [General]
I'm training a KNN model. If I want to know which value the "number of neighbors" option is set to, the score of the model is the best. I manually set different values one by one and check the scores, but this is not intuitive. Is there a faster way to know the results? If so, how do I need to operate it?

**Comment by ajdapretnar** [comment]
No, unfortunately Orange doesn't have any parameter optimization algorithms in place.

**Comment by lin2000y** [comment]
I am not referring to the optimization scheme for parameters, but to get the corresponding comparison under different parameters in a more intuitive and simple way. My current processing method is to place multiple different similar models, set different parameters for each model, and then compare them through scores

**Reply by ajdapretnar** [reply]
Ah, I think I know what you mean. You are wondering whether there's a "meta kNN" widget, where you could set parameter ranges and output several different models, each with its own parameters? Is that right?

No, Orange doesn't have that either. If you can think of a good UI solution for the task, feel free to propose it as a feature request.
